### DISCUSSION TITLE
Possible to run against dir of torrents to search for locally data to reseed?

### DISCUSSION THREAD

**Discussion by manure-taverns07** [cross-seed/cross-seed#264] [Q&A] [answered]
I'm trying to find an easy way to re-seed a bunch of torrents that might have files renamed. 

Say I have all my downloads in `/downloads` and my torrents in `/torrents/`. I don't want to run this against jackett or any external indexers. I just want to scan my `/downloads/` folder and match it against files in `/torrents/`, injecting the matches to my client (qbittorrent) and matching partial downloads if possible (e.g. a set of TV episodes where I only downloaded some of the episodes, I want to be able to re-seed just the files I originally downloaded).

Is this possible with cross-seed or is there a better tool you know about for the purpose?

**Comment by mmgoodnow** [comment, marked as answer]
auto torrent v2 might be able to help you out!

**Reply by manure-taverns07** [reply]
Thanks, yeah I had been playing with at2 but wasn't sure if it was possible to do a partial re-seed with it? I can check with the maintainer there if no-one here is sure
